staticjinja/dep_graph.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding:utf-8 -*-
+
+"""
+Dependency graph for staticjina
+"""
+
+from copy import deepcopy
+
+
+class DepGraph(object):
+    """
+    A directed graph which will handle dependencies between templates and data
+    files in a site.
+
+    :param site:
+        A :class:`Site <Site>` object.
+
+    """
+    def __init__(self, site):
+        self.parents = dict(
+                (s, set()) for s in site.jinja_names + site.data_names
+                )
```

Comment:
Doesn't this means dropping python 2.6 compatibility? Python 2.6 is the only reason I didn't use it. I must say I don't understand much to these python version stuff. I started python when python3 was already there and the first time I used tox was two weeks ago when preparing my preceding PR.